src/cli/cmds/help.ts

```diff
@@ -14,6 +14,57 @@
  *  limitations under the License.
  */
 
-export async function help(): Promise<void> {
+import Log from '../../lib/Log';
+import {ParsedArgs} from 'minimist';
+
+const HELP_MESSAGES = new Map<string, string>(
+    [
+      ['main', [
+        'llama-pack [command] <options>',
```

Comment:
You can use template literals for multiline strings.

Reply:
(if you want, I'm also fine with this way.)

Reply:
Tired it out, but it kind of breaks the indentation, as otherwise the blank characters at the beginning of the line are printed.

Reply:
Oh yeah, I'd have it formatted something like:

```
const HELP_MESSAGES = new Map<string, string> (
    [
      ['main',
`
llama-pack [command] <options>

build ............... generates an Android APK from a TWA Project
help ................ shows this menu
`],
      ['init',
`
...
`
```

But yeah, I'm fine with your way. (Mixing the indent of code and output strings is usually horrible - I guess the "proper" way to do this would be to have the help strings nicely formatted in text files that we read in (and it would allow for easier internationalisation, if we ever get that far!).